research/backend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`.vscode` is your own preference file for Visual Studio Code. You need to add `.vscode` to `.gitignore`; This code may influence other developer's code.

Reply:
Okay, I fixed it!